### ISSUE TITLE
Is Mixup necessary for MAE fine-tuning?

### ISSUE THREAD

**Issue by SCU-zly** [pengzhiliang/MAE-pytorch#99]
Your work is so excellent! However, when I try to fine-tuning on my downstream regression task, I found that when using Mixup method, the parameter of the function contains: `num_classes=args.nb_classes`.  Due to the special nature of my task, I have the output target of only one class.  So there's two possible solution for me:

- Set the num_classes to **1** naively.
- Disable the Mixup method.
_Besides, if I use Mixup, the loss function for my regression task should also find a proper one._
I'm confuse to use which solution or other better solution....I'm looking forward to your answer!
